release_notes/ocp-4-3-release-notes.adoc

```diff
@@ -275,6 +270,23 @@ The Samples Operator automatically recognizes the cluster architecture during
 installation and does not install incompatible x86_64 content on Power and Z
 architectures.
 
+The Samples Operator also uses Prometheus metrics to gather information about
+which imagestreams have failed to import, and if the Samples Operator has
+invalid configurations. An alert is sent to notify the administrator.
+
+[id="ocp-4-3-image-registry-operator"]
+=== Image Registry Operator
+
+The following enhancements are now available for the Image Registry Operator:
+
+* The registry management state is set as `Removed` on Baremetal, vSphere, and
+Red Hat Virtualization platforms so other storage providers can be configured.
+New installations must set the registry state to `Managed` in addition to
+provisioning storage.
+
+* If the registry's storage has changed, which could result in possible data
```

Comment:
I prefer front-loading the enhancement, something like: "An alert is sent to help prevent possible data loss when the registry storage has changed"